### DISCUSSION TITLE
Conecting piezos direclty rather than using audio jacks/connectors

### DISCUSSION THREAD

**Discussion by karanbhaikocodingnahiati** [corrados/edrumulus#151] [Q&A]
I'm thinking of connecting my piezo sensors directly to my analog front-end circuit. I'm confused if I should use some extra components like a new resistor/capacitor or maybe remove some of them since the old circuit given in prototype 2 uses audio jacks.

I'm using the prototype 2 circuit. Can anyone help me and give me a fresh new circuit using Piezos directly?

**Comment by corrados** [comment]
I see no issues in connecting the piezos directly without using the audio jacks. Simply replace the audio jacks by your piezos.

**Reply by karanbhaikocodingnahiati** [reply]
Ohh okay

I want to ask another question, I'm using my laptop to power ESP32, and the circuit has a 3.3V battery, is the new circuit going to be different if I use my laptop as a power source 

(I am new to electronics and this is my first project)

I have attached an image, kindly explain to me using an actual esp32 in the circuit diagram :)

Thank you soooo much for helping me out <3



![Screenshot 2024-10-16 212627](https://github.com/user-attachments/assets/843c42a2-8d17-4dda-b680-ff305c4d7b6e)

**Reply by corrados** [reply]
I am not sure how your circuit should work. Why don't you use the one given on the home page?

**Reply by karanbhaikocodingnahiati** [reply]
> I am not sure how your circuit should work. Why don't you use the one given on the home page?

ohkk, I'm gonna try that and will let you know if it worked or not!!!
